micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/UserProfile.js

```
@@ -381,7 +510,15 @@ const UserProfile = ({ stateCode, userType, cityDetails }) => {
               src={!profileImg || profileImg === "" ? defaultImage : profileImg}
             />
-            <button style={{ position: "absolute", left: "50%", bottom: "-24px", transform: "translateX(-50%)" }} onClick={onClickAddPic}>
+            <button
+              style={{
+                position: "absolute",
+                left: "50%",
+                bottom: "-24px",
+                transform: "translateX(-50%)",
+              }}
+              onClick={onClickAddPic}
+            >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)